wgsl/index.bs

```diff
@@ -4048,70 +4268,17 @@ WGSL defines the following attributes that can be applied to global variables:
   </xmp>
```

Comment:
Can we move the comment for `// A uniform buffer` to be above the `@group` to match the two below?